raglweb/__init__.py

```diff
@@ -288,7 +288,30 @@ def info():
 @app.route('/replay/<replay_hash>')
 def replay(replay_hash):
     db = _db_get()
-    cur = db.execute('SELECT filename FROM outcomes WHERE hash=:hash', dict(hash=replay_hash))
-    fullpath = cur.fetchone()['filename']
+    cur = db.execute('''
+        SELECT
+            filename,
+            p0.division as p0_division,
+            p1.division as p1_division,
+        FROM outcomes o
+        LEFT JOIN players p0 ON p0.profile_id = o.profile_id0
+        LEFT JOIN players p1 ON p1.profile_id = o.profile_id1
+        WHERE hash=:hash
+    ''', dict(hash=replay_hash))
+    row = cur.fetchone()
+
+    prefix = 'RAGL-S10-'
+    p0_division = row['p0_division']
+    p1_division = row['p1_division']
+    if p0_division == p1_division:
```

Comment:
This condition shouldn't be necessary: `ragl.py` does not record the game if the players are not in the same division. You can replace it with an `assert`.